protocol/x/clob/keeper/untriggered_conditional_orders.go

```diff
@@ -215,6 +216,10 @@ func (untriggeredOrders *UntriggeredConditionalOrders) RemoveUntriggeredConditio
 		}
 	}
 	untriggeredOrders.OrdersToTriggerWhenOraclePriceGTETriggerPrice = newOrdersToTriggerWhenOraclePriceGTETriggerPrice
+
```

Comment:
should funcs like these have timing stats? since they're now heap-sorting all conditional orders (although it should be fast since in-memory)